dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/validation/hooks/PreCheckSecurityOwnershipValidationHook.java

```diff
@@ -330,7 +330,9 @@ private void validateCreateEvent( ValidationErrorReporter reporter, Event event,
 
         checkEventWriteAccess( reporter, event, programStage, organisationUnit, ownerOrgUnit,
             categoryOptionCombo,
-            teiUid, isCreatableInSearchScope ); // TODO: calculate correct
+            teiUid, isCreatableInSearchScope, reporter.getBundle().getStrategy( event ) ); // TODO:
```

Comment:
reporter is already passed in, so we could save adding yet another param 😅

Reply:
Good point! :D